src/nectarchain/utils/stats.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Code that implement Welford's algorithm for stats computation
+
+This is inspired by the implementation done at https://github.com/a-mitani/welford
```

Comment:
I am checking the algorithm against the discussion found on https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance.
The implementation seems sound.